### DISCUSSION TITLE
Query and extract answers via api

### DISCUSSION THREAD

**Discussion by SwamiKannan** [searxng/searxng#3747] [Q&A] [answered]
I am trying to create an interface that can perform multiple functions - one of which is search. Hence, is it possible to use Searxng without the frontend ? i.e. send a query to my Searxng server (local) via api and return a Json object with all the details that are normally shown in the front end ? I searched for "no frontend" and "api" but most of the results for api were about the search apis. Thanks !

**Comment by vojkovic** [comment, marked as answer]
https://docs.searxng.org/admin/settings/settings_search.html

Add json to `formats:`

**Comment by SwamiKannan** [comment]
Thanks a lot !
